examples/deployment-chain/simple/app.pipecd.yaml

```diff
@@ -10,13 +10,13 @@ spec:
   postSync:
     chain:
       applications:
-        # The below will match application named `simple` of kind `CLOUDRUN`.
+        # The below will match application named `chain-simple-cloudrun` of kind `CLOUDRUN`.
         # which means the app that it's configuration is at
-        # https://github.com/pipe-cd/examples/blob/master/cloudrun/simple
-        - name: simple
+        # https://github.com/pipe-cd/examples/blob/master/deployment-chain/simple/chain-simple-cloudrun
+        - name: chain-simple-cloudrun
           kind: CLOUDRUN
-        # The below will match application named `simple` of kind `LAMBDA`.
+        # The below will match application named `chain-simple-lambda` of kind `LAMBDA`.
         # which means the app that it's configuration is at
-        # https://github.com/pipe-cd/examples/tree/master/lambda/simple
-        - name: simple
+        # https://github.com/pipe-cd/examples/tree/master/deployment-chain/simple/chain-simple-lambda
```

Comment:
Putting all in-chain applications inside the application directory of the first one can lead to misunderstandings by our users.
They might think that only ones in that directory could be used in the chain.
So How about putting them at the same directory level?

Reply:
Sure, I don't have a strong opinion on this 👍